File: app/src/main/java/com/dicoding/moviecataloguerv/data/source/model/Season.java
package com.dicoding.moviecataloguerv.data.source.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import lombok.Data;

@Data
public class Season {
    @Expose
    private int id;

    @Expose
    private String name;

    @SerializedName("air_date")
    @Expose
    private String airDate;

    @Expose
    private String overview;

    @Expose
    private List<Episode> episodes;

    @SerializedName("episode_count")
    @Expose
    private int totalEpisode;

    @SerializedName("poster_path")
    @Expose
    private String posterPath;

    @SerializedName("season_number")
    @Expose
    private int seasonNumber;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/model/Season.kt
package com.ryanrvldo.moviecatalogue.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Season(
    @Expose val id: Int,
    @Expose val name: String,
    @Expose @SerializedName("air_date") val airDate: String,
    @Expose val overview: String,
    @Expose val episodes: List<Episode>,
    @Expose @SerializedName("episode_count") val totalEpisodes: Int,
    @Expose @SerializedName("poster_path") val posterPath: String,
    @Expose @SerializedName("season_number") val number: Int
)